tools/cellprofiler/macros.xml

```diff
@@ -2,72 +2,104 @@
     <token name="@CP_VERSION@">3.1.9</token>
     <token name="@PY_VERSION@">2.7.16</token>
     <token name="@FORMATS@">jpg,png,tiff,bmp,gif,pcx,ppm,psd,pbm,pgm,eps</token>
+    <token name="@SPACES@">"    "</token>
+    <!-- four spaces needed for CP pipeline file -->
+    <token name="@COMMON_HELP@">
+        .. class:: infomark
+
+        **Input**
+
+        Existing CellProfiler pipeline file *(.cppipe)* or generated by linking CellProfiler tools.
+
+        .. class:: infomark
+
+        **Output**
+
+        The input CellProfiler pipeline file *(.cppipe)* including the settings of this module.
+
+        .. class:: warningmark
+
+        **IMPORTANT**
+
+        The first tool in a CellProfiler workflow has to be **Starting modules** and the last one **CellProfiler**.
```

Comment:
>  The first tool in a CellProfiler workflow has to be **Starting modules** and the last one **CellProfiler**.

 The first tool in a CellProfiler workflow has to be **Starting modules**. You execute the entire pipeline with the final **CellProfiler** tool, in which you feed in the images you want to process as well.